### ISSUE TITLE
`npm run web` has an error because it points to mongo. It must be postgres.

### ISSUE THREAD

**Issue by mosamu** [noobaa/noobaa-core#7299]
### Environment info
- NooBaa Version: *guy-p6* https://github.com/guymguym/noobaa-core/tree/guy-p6
- Platform: **standalone**

### Actual behavior
`npm run web` has the following error.
I guess `postgres` must be pointed out instead of `mongo`
```
May-16 11:59:32.579 [WebServer/271156]  [WARN] CONSOLE:: Error on reading supervisor file [Error: ENOENT: no such file or directory, stat '/data$
noobaa_supervisor.conf'] { errno: -2, code: 'ENOENT', syscall: 'stat', path: '/data/noobaa_supervisor.conf' }                                    
(node:271156) UnhandledPromiseRejectionWarning: Error: ENOENT: no such file or directory, stat '/data/noobaa_supervisor.conf'                    
(node:271156) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async functio$
 without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise reject$
on, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 2)
May-16 11:59:32.581 [WebServer/271156]   [LOG] CONSOLE:: read_rand_seed: got 32 bytes from /dev/random, total 32 ...
May-16 11:59:32.581 [WebServer/271156]   [LOG] CONSOLE:: read_rand_seed: closing fd ...
May-16 11:59:32.585 [WebServer/271156] [ERROR] core.util.mongo_client:: _connect: initial connect failed, will retry MongoNetworkError: failed t$
 connect to server [localhost:27017] on first connect [Error: connect ECONNREFUSED ::1:27017
    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1300:16) {
  name: 'MongoNetworkError'
}]
    at Pool.<anonymous> (/home/noobaa-core-guy-p6/node_modules/mongodb/lib/core/topologies/server.js:441:11)
    at Pool.emit (node:events:513:28)
    at Pool.emit (node:domain:489:12)
    at /home/noobaa-core-guy-p6/node_modules/mongodb/lib/core/connection/pool.js:564:14
    at /home/noobaa-core-guy-p6/node_modules/mongodb/lib/core/connection/pool.js:1000:11
    at /home/noobaa-core-guy-p6/node_modules/mongodb/lib/core/connection/connect.js:32:7
    at callback (/home/noobaa-core-guy-p6/node_modules/mongodb/lib/core/connection/connect.js:300:5)
    at Socket.<anonymous> (/home/noobaa-core-guy-p6/node_modules/mongodb/lib/core/connection/connect.js:330:7)
    at Object.onceWrapper (node:events:628:26)
    at Socket.emit (node:events:513:28)
```

### Expected behavior
1. `npm run web` completes.

### Steps to reproduce
1. `guy-p6` can produce this issue

### More information - Screenshots / Logs / Other output
This is a standalone machine,

**Comment by martin-petermann** [comment]
Just as a work around I added the following line to the ```config-local.js``` file:
```yaml
config.DB_TYPE = 'postgres';
```

**Comment by guymguym** [comment]
fixed by #7296
